script/audit_deps.py

```diff
@@ -27,7 +27,7 @@
 IGNORED_CARGO_ADVISORIES = [
     # Remove when:
     # https://github.com/chronotope/chrono/issues/602 is resolved
-    # Tracking issue: https://github.com/brave/brave-browser/issues/18838
+    # Tracking issue: https://github.com/brave/brave-browser/issues/20459
```

Comment:
Thanks for updating the bug reference!

Reply:
Updating this to https://github.com/brave/brave-browser/issues/20568 before I merge